### PR TITLE
docs: add 'Key Topics' section to tutorials

### PR DESCRIPTION
**Pull request by Classy-Bear**
## Status

**READY**

## Breaking Changes

NO

## Description

This PR is about the https://github.com/felangel/bloc/issues/1796 issue, before adding another  'What's the point?' section in other tutorials see how it will look like:

<details>
<summary>Photo</summary>

Screen Shot 2021-02-05 at 7 20 10 PM
</details>

* Explain brevely what BLoC concepts the tutorial will be about
* Use links to refer to this concepts
* Indicate new best practices with ⚡ 

Benefits? UX and shows the concepts that has to be learned (inspiration came from [here](https://dart.dev/tutorials/language/streams) in 'What's the point' section)

*It also adds the v6.0.1 BLoC core concepts to the spanish docs.*

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/es/README.md [hunk 1]
```diff
@@ -162,3 +173,7 @@ Obtenga más información en los siguientes enlaces, que han sido aportados por
 ## Mantenedores
 
 - [Felix Angelov](https://github.com/felangel)
+
+## Partidarios
+
+[<img src="https://raw.githubusercontent.com/felangel/bloc/master/docs/assets/vgv_logo.png" width="120" />](https://verygood.ventures)
```
Comment (felangel) [inline docs/es/README.md:179]: super minor but I've updated the README to move the "Sponsors" section higher up and to include stream. Can we adjust this to be consistent with https://github.com/felangel/bloc/blob/master/README.md? Thanks! 🙏

**Review by felangel — COMMENTED** [review]
Thanks so much for taking the time to make these changes! I really appreciate it! 🙏 💙 
Let me know when you'd like me to re-review and merge these changes 👍

File: docs/es/README.md (hunk 1 above)
Comment (Classy-Bear) [reply docs/es/README.md:179]: Sure, no problem!

**Comment by Classy-Bear** [comment]
Yeah sure, happy to help :) . This PR will take a while because I will be also updating the /es/ documentation or should I do it later and finish with this issue first? 

The Spanish doc is so close to be completed, I think an Angular tutorial remains to be translated.

**Comment by felangel** [comment]
> Yeah sure, happy to help :) . This PR will take a while because I will be also updating the /es/ documentation or should I do it later and finish with this issue first?
> 
> The Spanish doc is so close to be completed, I think an Angular tutorial remains to be translated.

I think it would be better to have smaller PRs which we can merge quickly 👍

File: docs/flutterangulargithubsearch.md [hunk 2]
```diff
@@ -8,6 +8,14 @@
 
 ![demo](./assets/gifs/angular_github_search.gif)
 
+## What's the point?
+
+- [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), Flutter widget which provides a bloc to its children.
+- [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), Flutter widget that handles building the widget in response to new states.
+- Using Bloc instead of Cubit. [Want to know the difference?](/coreconcepts?id=cubit-vs-bloc)
+- Prevent unnecessary rebuilds with [Equatable](/faqs?id=when-to-use-equatable).
+- Use the `transformEvents` method with Redux.
```
Comment (felangel) [inline docs/flutterangulargithubsearch.md:17]:
```suggestion
- Use the `transformEvents` method with Rx.
```

File: docs/flutterinfinitelisttutorial.md [hunk 3]
```diff
@@ -6,6 +6,16 @@
...
 
+## What's the point?
+
+- Observe state changes with [BlocObserver](/coreconcepts?id=blocobserver).
+- [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), Flutter widget which provides a bloc to its children.
+- [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), Flutter widget that handles building the widget in response to new states.
+- Using Cubit instead of Bloc. [Want to know the difference?](/coreconcepts?id=cubit-vs-bloc)
+- Adding events with [context.read](/migration?id=❗contextbloc-and-contextrepository-are-deprecated-in-favor-of-contextread-and-contextwatch).⚡
+- Prevent unnecessary rebuilds with [Equatable](/faqs?id=when-to-use-equatable).
+- Use the `transformEvents` method with Redux.
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:17]:
```suggestion
- Use the `transformEvents` method with Rx.
```

File: docs/fluttercountertutorial.md [hunk 4]
```diff
@@ -6,6 +6,13 @@
 
 ![demo](./assets/gifs/flutter_counter.gif)
 
+## What's the point?
+- Observe state changes with [BlocObserver](/coreconcepts?id=blocobserver).
```
Comment (felangel) [inline docs/fluttercountertutorial.md:10]:
```suggestion

- Observe state changes with [BlocObserver](/coreconcepts?id=blocobserver).
```

File: docs/angularcountertutorial.md [hunk 5]
```diff
@@ -6,6 +6,10 @@
 
 ![demo](./assets/gifs/angular_counter.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/angularcountertutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/flutterangulargithubsearch.md [hunk 6]
```diff
@@ -8,6 +8,14 @@
 
 ![demo](./assets/gifs/angular_github_search.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/flutterangulargithubsearch.md:11]:
```suggestion
## Key Topics
```

File: docs/fluttercountertutorial.md [hunk 7]
```diff
@@ -6,6 +6,14 @@
 
 ![demo](./assets/gifs/flutter_counter.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/fluttercountertutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/flutterfirebaselogintutorial.md [hunk 8]
```diff
@@ -6,6 +6,17 @@
 
 ![demo](./assets/gifs/flutter_firebase_login.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/flutterfirebaselogintutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/flutterfirestoretodostutorial.md [hunk 9]
```diff
@@ -6,6 +6,16 @@
 
 ![demo](./assets/gifs/flutter_firestore_todos.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/flutterfirestoretodostutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/flutterinfinitelisttutorial.md [hunk 10]
```diff
@@ -6,6 +6,16 @@
 
 ![demo](./assets/gifs/flutter_infinite_list.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/flutterlogintutorial.md [hunk 11]
```diff
@@ -6,6 +6,17 @@
 
 ![demo](./assets/gifs/flutter_login.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/flutterlogintutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/fluttertimertutorial.md [hunk 12]
```diff
@@ -6,6 +6,16 @@
 
 ![demo](./assets/gifs/flutter_timer.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/fluttertimertutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/fluttertodostutorial.md [hunk 13]
```diff
@@ -6,6 +6,15 @@
 
 ![demo](./assets/gifs/flutter_todos.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/fluttertodostutorial.md:9]:
```suggestion
## Key Topics
```

File: docs/flutterweathertutorial.md [hunk 14]
```diff
@@ -6,6 +6,18 @@
 
 ![demo](./assets/gifs/flutter_weather.gif)
 
+## What's the point?
```
Comment (felangel) [inline docs/flutterweathertutorial.md:9]:
```suggestion
## Key Topics
```

**Review by felangel — APPROVED** [review]
Thanks so much! I really appreciate the amazing contributions! 💯 💙 🚀

**Pull request merged**